internal/kms/kes.go

```diff
@@ -346,34 +338,23 @@ func (c *kesClient) DecryptAll(ctx context.Context, keyID string, ciphertexts []
 	return plaintexts, nil
 }
 
-// DescribePolicy describes a policy by returning its metadata.
-// e.g. who created the policy at which point in time.
-func (c *kesClient) DescribePolicy(ctx context.Context, policy string) (*kes.PolicyInfo, error) {
-	c.lock.RLock()
-	defer c.lock.RUnlock()
-
-	return c.enclave.DescribePolicy(ctx, policy)
-}
-
-// AssignPolicy assigns a policy to an identity.
-// An identity can have at most one policy while the same policy can be assigned to multiple identities.
-// The assigned policy defines which API calls this identity can perform.
-// It's not possible to assign a policy to the admin identity.
-// Further, an identity cannot assign a policy to itself.
-func (c *kesClient) AssignPolicy(ctx context.Context, policy, identity string) error {
+// DecryptKey decrypts the ciphertext with the key at the KES
```

Comment:
The function description does not match the function name

Reply:
👍